src/ast2ram/seminaive/ClauseTranslator.cpp

```diff
@@ -621,6 +688,12 @@ Own<ram::Condition> ClauseTranslator::getFunctionalDependencies(const ast::Claus
 }
 
 std::vector<ast::Atom*> ClauseTranslator::getAtomOrdering(const ast::Clause& clause) const {
+    // set dominating/dominated head of clause
+    if (isA<ast::SubsumptiveClause>(clause)) {
```

Comment:
Minor: Is this clause needed, isn't it already set to the same values [here](https://github.com/souffle-lang/souffle/pull/2114/files#diff-3b0da7e61fdee8b886a3ac19347584c62c3e847072455e9d5c4e6f56e74e54bbR138)?